apps/coscrad-frontend-e2e/setup-cypress-environment.sh

```diff
@@ -0,0 +1,24 @@
+echo "Setting Environment..."
+export NODE_ENV="e2e"
+
+FILE="e2e.env"
```

Comment:
Possibly call this file `cli.env.json` or add a comment at the beginning of the file that it is for the CLI and is generated automagically